lib/inferno/dsl/fhir_client.rb

```diff
@@ -60,26 +60,59 @@ def fhir_clients
         @fhir_clients ||= {}
       end
 
+      # Move parameters from body to the path for GET operations
```

Comment:
This description isn't quite right anymore, and one of the listed params doesn't exist anymore.